modules/c/nrt/source/DateTime.c

```diff
@@ -855,6 +927,26 @@ NRTPRIV(char *) _NRT_strptime(const char *buf, const char *fmt, struct tm *tm,
 
     }
 
+    /* If we the day of year and year, infer the corresponding month
```

Comment:
Missing "have" (should be "If we have the day of the year").

Reply:
Fixed